datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/scripts/github_user.py

```diff
@@ -0,0 +1,39 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import click
+import requests
+
+from ...console import CONTEXT_SETTINGS, abort, echo_success
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Lookup Github username by email.')
+@click.argument('email')
+def github_user(email):
+    """Given an email, attempt to find a Github username
+       associated with the email.
+
+    `$ ddev meta scripts github-user example@datadoghq.com`
+    """
+    usernames = []
```

Comment:
Multiple?

Reply:
Yeah it's probably not necessary to handle multiple